docs/README.md

```diff
@@ -75,14 +96,12 @@ Any and all help is greatly appreciated! Check out our page on [how you can help
 
 ## Roadmap (What's on our mind)
 
+- [ ] Better DX for Mirascope CLI (e.g. autocomplete)
```

Comment:
omit, this will be in the release

Reply:
omit or mark as a check like I did for functions as OpenAI tools?

Reply:
i think check is fine until the list gets too long e.g. more than 5 items, then we can prune the checked ones